doc/classes/AStarGrid2D.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AStarGrid2D" inherits="RefCounted" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		An implementation of A* to find the shortest paths among the 2D grid.
```

Comment:
```suggestion
		A* (or "A-Star") pathfinding tailored to find the shortest paths on 2D grids.
```
Thoughts:
- "implementation" is filler
- A* should be explained as "A-Star" for SEO and new people who don't know how its pronounced yet
- "tailored" tries to convey it is a specialized form; ideally, I'd like to point to the generalized `[AStar2D]` here but it gets long then
- mentioning "pathfinding" as its the common name/concept people may be aware of
- the "among the 2D grid" felt a bit clunky, hope this is better?

Reply:
You know better how it sounds - I'm not a native English speaker :)